salt/modules/status.py

```diff
@@ -69,21 +69,21 @@ def _get_boot_time_aix():
     case $t in *-*) d=${t%%-*}; t=${t#*-};; esac
     case $t in *:*:*) h=${t%%:*}; t=${t#*:};; esac
     s=$((d*86400 + h*3600 + ${t%%:*}*60 + ${t#*:}))
-
-    t is 7-20:46:46
     """
-    boot_secs = 0
     res = __salt__["cmd.run_all"]("ps -o etime= -p 1")
     if res["retcode"] > 0:
         raise CommandExecutionError("Unable to find boot_time for pid 1.")
     bt_time = res["stdout"]
-    days = bt_time.split("-")
-    hms = days[1].split(":")
+    match = re.match(r"(?:(\d+)-)?(?:(\d\d):)?(\d\d):(\d\d)", bt_time)
```

Comment:
Could you maybe consider using named capture groups? Ex:
```
re.match(r"(?:(?P<days>\d+)-)?((?P<hours>\d\d):)?(?P<minutes>\d\d):(?P<seconds>\d\d)", '7-20:46:46').group('hours')
```

Reply:
Could do, but it seemed a bit overkill.

Reply:
fine as is